src/simularium/LocalFileSimulator.ts

```diff
@@ -133,4 +140,45 @@ export class LocalFileSimulator implements ISimulator {
     public getSimulariumFile(): ISimulariumFile {
         return this.simulariumFile;
     }
+
+    public async setupMetricsCalculator(
```

Comment:
Nit: Consider `setMetricsCalculator`? I know this will eventually do setup steps, but it makes it clear that it's setting a local field in addition to whatever other setup it'll be doing. Also `setup` implies to me that it would be returning the modified `RemoteMetricsCalculator`